### PR TITLE
React integration

### PR DESCRIPTION
**Pull request by bbaksh**
npm run dev

TODO:
- Make configurations better
- Add hot-reload and other useful dev settings

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var path = require('path');
```
Comment (JoeRoussy) [inline webpack.config.js:1]: Can we use es6 for consistency?

Example:
`import path from 'path';`

`export default ...`

File: app/server.js [hunk 2]
```diff
@@ -60,8 +64,11 @@ const dbLogger = getChildLogger({
 
 
     // TODO: Configure actual routes here:
+    // SIDENOTE: Routes can be done with react I believe.
+    // We can create our main architecture through this component
     app.get('/', (req, res) => {
-        res.send('Hello world!');
+        var html = ReactDOMServer.renderToString(React.createElement(Routes));
```
Comment (JoeRoussy) [inline app/server.js]: `const html =`

Comment (bbaksh) [reply app/server.js]: I'm not sure we can use const here. Perhaps 'let' might be the way to go.

File: webpack.config.js (hunk 1 above)
Comment (bbaksh) [reply webpack.config.js:1]: I think I tried using es6 in that file and it was complaining. I'll get a SS later on when i'm on my mac, maybe you know the solution

File: app/server.js (hunk 2 above)
Comment (JoeRoussy) [reply app/server.js]: I'm pretty sure can use const. const refers to immutability of the reference not the value. 

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

**Pull request merged**